db.js

```diff
@@ -0,0 +1,113 @@
+const Sequelize = require('sequelize');
+const {STRING, UUID, UUIDV4, DataTypes} = Sequelize;
+const conn = new Sequelize(process.env.DATABASE_URL || 'postgres://localhost/grace_shopper_db');
+
+const User = conn.define('user', {
+  id: {
+    primaryKey: true,
+    type: UUID,
+    defaultValue: UUIDV4
+  },
+  name: {
+    type: STRING,
+    allowNull: false
+  },
+  email: {
+    type: STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+      isEmail: true
+    }
+  },
+  password: {
+    type: STRING,
+    allowNull: false
+  }
+})
+
+const Product = conn.define('product', {
+  id: {
+    primaryKey: true,
+    type: UUID,
+    defaultValue: UUIDV4
+  },
+  name: {
+    type: STRING,
+    allowNull: false
+  },
+  price: {
+    type: DataTypes.DECIMAL(10,2),
+    validate: {
+      isDecimal: true
+    }
+  }
+});
+
+Product.belongsTo(User)
+User.hasMany(Product);
+
+const syncAndSeed = async ()=>{
+  await conn.sync({force: true});
```

Comment:
`force: true` can get you in trouble with deployed apps - this means when you next deploy your app, it will delete all user records (e.g. purchases, new sign ups, etc).